man/man1/oci-kvm.1

```diff
@@ -140,6 +202,12 @@ oci-kvm create --disk-size 128 --pool guest_pool --net 10.0.0.64 -D foo  --virt
 .RS 3
 Creates a guest with a root disk of 128GB on storage pool named foo
 .RE
+
+oci-kvm create-network --network-name=mynet0 --net=10.0.1.2 --vcn-cidr=10.0.0.0/16 --ip-first=10.0.2.20 --ip-start=10.0.2.10 --ip-end=10.0.2.30 --ip-prefix=24
+.RS 3
+Creates virtual network named mynet0 on vNIC with 10.0.1.2 address
```

Comment:
Creates a virtual network named mynet0 on the vNIC that has an ip address of 10.0.1.2

On a side not, you might want to use a completely different IP network for the guest network to make this clearer, something like oci-kvm create-network --network-name=mynet0 --net=10.0.1.2 --vcn-cidr=10.0.0.0/16 --ip-first=192.168.10.20 --ip-start=192.168.10.10 --ip-end=192.168.10.30 --ip-prefix=24

But even this is wrong since the bridge IP should not be inside the DHCP range

Reply:
changed to 
--ip-bridge=192.168.10.9 --ip-start=192.168.10.10 --ip-end=192.168.10.30